Cargo.toml

```diff
@@ -21,10 +21,15 @@ keywords = ["http"]
 categories = ["web-programming"]
 edition = "2018"
 
+[features]
+default = []
+url = ["url_2"]
```

Comment:
I'd probably just have the feature called `url_2` which is already implied by the optional dependency.

Reply:
Errors out with: `Features and dependencies cannot have the same name: `url_2``. The two need to be different names.

Reply:
Optional dependencies are automatically features, you should just be able to remove the features block entirely.

Reply:
Oh, neat. I didn't realize that. Fixing.

Reply:
done.